Update Gallery with new Cookbook

### Root Repository Name

interactive-sentinel-2-cookbook

### Did you check if your contribution would be better suited as an addition to an existing Cookbook, before generating a new one?

yes

### Additional comments

This Project Pythia Cookbook provides a recipe for building an interactive dashboard for the Sentinel-2 L2A satellite imagery using the holoviews ecosystem.

Unable to validate input
```
Run python .github/workflows/collect-user-submission.py
Traceback (most recent call last):
  File ".github/workflows/collect-user-submission.py", line 38, in <module>
    issue = IssueInfo(gh_event_path=os.environ["GITHUB_EVENT_PATH"]).create_submission()
  File ".github/workflows/collect-user-submission.py", line 23, in create_submission
    self._create_submission_input()
  File ".github/workflows/collect-user-submission.py", line 27, in _create_submission_input
    text = self.data["issue"]["body"]
AttributeError: 'IssueInfo' object has no attribute 'data'
Error: Process completed with exit code 1.
```

Sorry I will look into this bug for you, in the mean time I've created a PR for you in #166

--- a/site/cookbook_gallery.txt
+++ b/site/cookbook_gallery.txt
@@ -8,3 +8,4 @@ kerchunk-cookbook
 xbatcher-ML-1-cookbook
 dask-cookbook
 ERA5_interactive-cookbook
+interactive-sentinel-2-cookbook

